File: Python/Program/Aksesoris.py
from PetShop import PetShop


class Aksesoris(PetShop):
    def __init__(self, id, nama_produk="", harga_produk=0.0, stok_produk=0, jenis="", bahan="", warna=""):
        super().__init__(id, nama_produk, harga_produk, stok_produk)
        self.jenis = jenis
        self.bahan = bahan
        self.warna = warna

    # getters
    def get_jenis(self):
        return self.jenis

    def get_bahan(self):
        return self.bahan

    def get_warna(self):
        return self.warna

    # setters
    def set_jenis(self, jenis):
        self.jenis = jenis

    def set_bahan(self, bahan):
        self.bahan = bahan

    def set_warna(self, warna):
        self.warna = warna

File: Java/Program/Aksesoris.java
class Aksesoris extends PetShop {
    protected String jenis;
    protected String bahan;
    protected String warna;

    // constructors
    public Aksesoris() {
        super();
        this.jenis = "";
        this.bahan = "";
        this.warna = "";
    }

    public Aksesoris(int id, String namaProduk, double hargaProduk, int stokProduk, String jenis, String bahan,
            String warna) {
        super(id, namaProduk, hargaProduk, stokProduk);
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    // getters
    public String getJenis() {
        return jenis;
    }

    public String getBahan() {
        return bahan;
    }

    public String getWarna() {
        return warna;
    }

    // setters
    public void setJenis(String jenis) {
        this.jenis = jenis;
    }

    public void setBahan(String bahan) {
        this.bahan = bahan;
    }

    public void setWarna(String warna) {
        this.warna = warna;
    }
}
